src/c/admin-contribution-user.js

```diff
@@ -0,0 +1,16 @@
+window.c.AdminContributionUser = (function(m) {
```

Comment:
Let's add a doctoring explaining the component here.
For an example you can take a look here: https://github.com/catarse/catarse.js/blob/a0ac0c21ba6820456628b81e46d3f06a1e55e951/src/c/admin-external-action.js#L1